website/content/docs/platform/k8s/helm/run.mdx

```diff
@@ -441,6 +441,15 @@ Vault has K8s service discovery built in (when enabled in the server configurati
 will automatically change the labels of the pod with its current leader status. These labels
 can be used to filter the pods.
 
+!> **IMPORTANT NOTE:** If `ha.raft` enabled with Vault 1.11+, Autopilot has the ability to do
+[Automated Upgrades](/docs/concepts/integrated-storage/autopilot#automated-upgrades).
+If using the default `server.updateStrategyType` of `OnDelete` for the Vault StatefulSet, then
+the same upgrade procedure below will continue to work as expected.
+As such, the Vault team highly recommends using `OnDelete` with Raft Autopilot automated upgrades.
+If overriding the strategy type to `RollingUpdate`, then ensure that there are sufficient cluster
+resources available for the rolling update or that the pod anti-affinity rules in
+`server.affinity` are adjusted.
+
 For example, select all pods that are Vault standbys:
```

Comment:
This "For example" goes with the label filtering text that preceded it, but now the reader might think that it refers to the "ensure that there are sufficient cluster resources" sentence.